genScript.js

```diff
@@ -0,0 +1,61 @@
+const services = require('./services')
+
+console.log('#!/bin/bash')
+console.log('set -ex')
```

Comment:
Is there a reason for this script to be node generating bash, rather than just a bash script?

Reply:
Alternatively, could we have the node script call the subprocesses directly? I'm sure there's a subprocess-sync interface in node somewhere, which should make it all relatively easy.

The other alternative would be python, or whatever scripting languages the base image has installed?

Reply:
As noted in the PR description: we need to fetch multiple details from the `services.js` file -- e.g. get `.repo` and `.name` field for cloning -- which is not straight forward unless we invoke node 100s of times and get one detail at a time (`REPO=$(node ...; NAME=$(node ...); git clone "$REPO" "$NAME"`).